From 8a7803d8c57ad39d05541ce6e6c0460eeae9da01 Mon Sep 17 00:00:00 2001
From: Gio <thelookouts@hotmail.com>
Date: Wed, 3 Aug 2022 04:26:03 -0400
Subject: [PATCH] Added DERE DBook D10 (#24173)

* Added DERE DBook D10

(cherry picked from commit 2d2efed2eeebad6860e9fa4880d02286175dcffa)

Related: #2087778
---
 hwdb.d/60-sensor.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index fc688ef4d3..1c3f6b62bb 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -339,6 +339,14 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnDellInc.*:pnVenue8Pro3845:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnDell*:pnVenue10Pro5055:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; 1, 0, 0; 0, 0, 1
 
+#########################################
+# DERE
+#########################################
+
+# DBook D10
+sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInternational,LLC.:bvrJP2V1.05:bd04/27/2022:br1.5:efr1.3:svnDefaultstring:*
+ ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
+
 #########################################
 # DEXP
 #########################################
